modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -1877,7 +1951,11 @@ static int icv_av_write_frame_FFMPEG( AVFormatContext * oc, AVStream * video_st,
     {
         /* encode the image */
 #if USE_AV_SEND_FRAME_API
-        ret = avcodec_send_frame(c, picture);
+        if (picture == NULL && frame_idx == 0) {
```

Comment:
@michal-kowalczyk Nice find! Is this change relevant for 4.5.x?

Reply:
I believe that message is addressed to @mikhail-nikolskiy .

Reply:
This feature patch is targeted to the master branch (4.x releases).

Probably this can be backported to 3.4.x after merging.

Reply:
This case is checked by `videoio/Videoio_Writer.write_nothing` test (available on both 3.4 and master branches)

`avcodec_send_frame()` API is not used on 3.4 branch, so backport is not applicable here.